packages/backend/package.json

```diff
@@ -3,16 +3,19 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "dev": "convex dev --tail-logs",
+    "dev": "convex dev --tail-logs --live-component-sources --typecheck-components",
     "setup": "convex dev --once && convex env set VALIDATE_ENV=true",
     "seed": "convex dev --once && convex run init"
   },
   "type": "module",
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@convex-dev/auth": "^0.0.74",
+    "@convex-dev/polar": "link:@convex-dev/polar",
```

Comment:
ah I think this could be why there's an issue with the http cast

Reply:
Ah yeah, it's still linked for now. And yeah, that was my thought as well, although the same code being used for both should have still avoided this issue (I think?)